common/src/ethereum.rs

```diff
@@ -18,12 +18,17 @@ use ethers::types::Chain;
 use rad_terminal::args;
 use rad_terminal::components as term;
 
+mod walletconnect;
+
+use self::walletconnect::WalletConnect;
```

Comment:
```suggestion
use walletconnect::WalletConnect;
```

Reply:
```
error[E0659]: `walletconnect` is ambiguous
  --> common/src/ethereum.rs:23:5
   |
23 | use walletconnect::WalletConnect;
   |     ^^^^^^^^^^^^^ ambiguous name
   |
   = note: ambiguous because of multiple potential import sources
   = note: `walletconnect` could refer to a crate passed with `--extern`
   = help: use `::walletconnect` to refer to this crate unambiguously
note: `walletconnect` could also refer to the module defined here
  --> common/src/ethereum.rs:21:1
   |
21 | mod walletconnect;
   | ^^^^^^^^^^^^^^^^^^
   = help: use `self::walletconnect` to refer to this module unambiguously
   ```

Reply:
Ah right, we are also importing a crate of the name, gotcha :)